speechbrain/pretrained/interfaces.py

```diff
@@ -3028,3 +3091,51 @@ def transcribe_batch(self, wavs, wav_lens):
     def forward(self, wavs, wav_lens):
         """Runs full transcription - note: no gradients through decoding"""
         return self.transcribe_batch(wavs, wav_lens)
+
```

Comment:
Shouldn't we also have a high level interface that combines everything and goes from audio to audio? This would be neat on HF hub.